doc/man1/cmds.pod

```diff
@@ -0,0 +1,151 @@
+=pod
+
+=head1 NAME
+
+asn1parse,
+ca,
+ciphers,
+cms,
+crl,
+crl2pkcs7,
+dgst,
+dhparam,
+dsa,
+dsaparam,
+ec,
+ecparam,
+enc,
+engine,
+errstr,
+gendsa,
+genpkey,
+genrsa,
+kdf,
```

Comment:
Remove kdf, mac,... oh, and storeutl from this list

Reply:
The idea was to have _no_ distinction between old and new commands, as @richsalz criticized. In 3.0, all unprefixed manual queries end up on the 'deprecation' page cmds.pod, and in 4.0, all unprefixed entries are gone.

Reply:
So we're adding to the global namespace clutter...  I disagree.

Reply:
(but if that's the bigger consensus, then you should add `info`)

Reply:
> So we're adding to the global namespace clutter... I disagree.

I'm not insisting. Still open for both variants.

Reply:
Currently slightly favoring consistency over no-cluttering. Because time users will see the new commands and try 'man kdf' first, because they are used to it.

So better we give them some opportunity to learn the new way ;-)